docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>Can you link information about the explanation of the sanboxes that we give in <code>getting_started</code> under <code>Generated Output</code>? I would not want to assume that users know about sandboxes when reading this tutorial. An issue is that in <code>getting_started</code> we introduce only the client and session sanboxes as it is an introduction. We should document the pilot and task sanboxes here (see a discussion with Andre on another PR).</p><p> </p><p>"to stage input data automatically" -&gt; "to stage the task's input data before its execution."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>resolved</p>